src/main/java/github/workflow/kata/Grid.java

```diff
@@ -6,10 +6,69 @@
  * A gaming grid.
  */
 public class Grid {
-    private final String gridDef;
+    private String gridDef;

```

Comment:
There are probably better data structures to represent a grid.

Reply:
would recommend using either a single array with a stride parameter or a 2xdimensional array

Reply:
going for 2xdim array, strides are a bit cumbersome.